test/remoting.integration.js

```diff
@@ -238,5 +208,154 @@ describe('remoting - integration', function() {
         ];
         expect(methods).to.include.members(expectedMethods);
       });
+
+    it('has expected remote methods with model.settings.replaceOnPUT set to true', function() {
+      app.models.store.settings.replaceOnPUT = true;
+      app.models.store.setup();
```

Comment:
@Amir-61 Based on your earlier comment, this is expected to not work correctly. Do you need to create a new model with `replaceOnPUT:true` too?